user/for-beginners.md

```diff
@@ -36,22 +36,21 @@ you can have jobs depend on each other with [Build Stages](/user/build-stages/),
 set up [notifications](/user/notifications/), prepare
 [deployments](/user/deployment/) after builds and many other tasks.
 
-## Builds, Jobs, Stages and Phases
+## Builds, Stages, Jobs and Phases
 
 In the Travis CI documentation, some common words have specific meanings:
 
-* *phase* - the [sequential steps](/user/job-lifecycle/)
-  of a job. For example, the `install` phase, comes before the `script` phase,
-  which comes before the optional `deploy` phase.
+* *build* - a group of *jobs* that run in sequence. For example, a build might have two *jobs*, each
```

Comment:
If `stages` are a group of jobs that run in parallel, it might be useful specifying that `builds` are groups of jobs that run in sequence (assuming this is true).